stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -178,9 +182,7 @@ public class MessageListView : ConstraintLayout, IMessageListView {
     private val DEFAULT_USER_CLICK_LISTENER = UserClickListener { /* Empty */ }
     private val DEFAULT_READ_STATE_CLICK_LISTENER =
         ReadStateClickListener { reads: List<ChannelUserRead> ->
-            ReadUsersDialog(context)
-                .setReads(reads)
-                .show()
```

Comment:
I assume the new design will have a new implementation of this, so I didn't want to make it shared